ingest/bulk_upload.py

```diff
@@ -56,9 +57,9 @@ def ICU_installed(es):
         except elasticsearch.exceptions.NotFoundError:
             debug("Index not found, continuing")
 
-    # Create an additional field used for sorting. The new field is called
-    # cf.sort and will use a locale-aware collation. We need to do this before
-    # ingesting the data, so that the new field is properly populated.
+    # Create two additional fields used for sorting. The new fields are called
+    # cf.sort and completions, they will use a locale-aware collation. We need to do this 
+    # before ingesting the data, so that the new field is properly populated.
```

Comment:
Technically only `cf.sort` uses the special collation for its text(that's the `"icu_collation_keyword"`). The `completions` use is of a different type specifically for completions.